hooks/basic/collector.py

```diff
@@ -921,22 +914,27 @@ def _resolve_item_fields(self, settings, item):
         return fields
 
 
-def _build_seq_extensions_list():
+def _build_seq_extensions_list(settings):
 
     file_types = ["file.%s" % x for x in KNOWN_IMAGE_TYPES]
     extensions = set()
-
+    defined_file_types = settings['Item Types'].keys()
+    # get defined_file_type based on file_types
     for file_type in file_types:
-        extensions.update(DEFAULT_ITEM_TYPES[file_type]["extensions"])
-
+        for defined_file_type in defined_file_types:
+            if defined_file_type.startswith(file_type):
+                extensions.update(settings['Item Types'][defined_file_type]["extensions"].value)
+                extensions.update(e_item.upper() for e_item in settings['Item Types'][defined_file_type]["extensions"].value)
+                break
     # get all the image mime type extensions as well
     mimetypes.init()
     types_map = mimetypes.types_map
     for (ext, mimetype) in types_map.iteritems():
         if mimetype.startswith("image/"):
             extensions.add(ext.lstrip("."))
+            extensions.add(ext.lstrip(".").upper())
 
     return list(extensions)
 
+
 KNOWN_IMAGE_TYPES = ("render", "texture", "image", "deep_render")
```

Comment:
Garvit suggests this should also be a setting.

Reply:
I will create a separate ticket for the setting.